src/test/lib/TestConnection.cpp

```diff
@@ -796,6 +796,18 @@ TestConnection::SetResumptionTicket(
             NewResumptionTicket->Buffer);
 }
 
+QUIC_STATUS
+TestConnection::SetQtipPreferences(
+    const uint8_t OnOrOff
+) const {
+    return
+        MsQuic->SetParam(
+            QuicConnection,
+            QUIC_PARAM_CONN_QTIP,
+            sizeof(uint8_t),
```

Comment:
nit: It is generally better to use `sizeof(variable)` rather than `sizeof(type)`, that way, they can't mismatch.

```suggestion
            sizeof(OnOrOff),
```

Reply:
ACK